pokemon-showdown

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env node
 'use strict';
 
+const spawnSync = require('child_process').spawnSync;
```

Comment:
Generally I prefer explicitly importing modules like this, even if I only use them once, in case in the future I might need to use them more. If it's that much of an issue, I can change it though